aspnetcore/signalr/introduction-signalr-core.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Introduction to SignalR on ASP.NET Core
+author: rachelappel
+description: Learn how the ASP.NET Core SignalR library simplifies adding real-time web functionality to apps.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 2/19/2018
+ms.prod: aspnet-core
+ms.technology: dotnet-signalr
+ms.topic: article
+uid: signalr/introduction-signalr-core
+---
+# Introduction to SignalR
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+## What is SignalR?
+
+ASP.NET Core SignalR is a library that simplifies adding real-time web functionality to apps. Real-time web functionality is the ability to have server code push content to connected clients instantly as it becomes available, rather than having the server wait for a client to request new data.
+
+Good candidates for adding real-time updates include:
+
+* Apps that require high frequency updates from the server. Real-time gaming, social networks, maps and GPS apps, require high frequency updates.
```

Comment:
something like:

Apps that require high frequency updates from the server, for example, Real-time gaming, social networks, maps and GPS apps, ~require high frequency updates.~